src/CMakeLists.txt

```diff
@@ -151,3 +151,4 @@ configure_file("systemd/cjet.service.in" "cjet.service" @ONLY)
 
 INSTALL(TARGETS  cjet RUNTIME DESTINATION bin)
 INSTALL(FILES "${CMAKE_CURRENT_BINARY_DIR}/cjet.service" DESTINATION lib/systemd/system)
+INSTALL(FILES "systemd/avahi/cjet.service" DESTINATION bin/avahi)
```

Comment:
I have another idea. avahi service files are placed within /etc/avahi/services but with another extension.
We install cjet.service.inactive and rename it if the service is started and rename it back when the service stops